src/components/CardSeparator.tsx

```diff
@@ -10,7 +11,27 @@ import classnames, {
   width,
 } from 'classnames/tailwind'
 
-type Color = 'accent' | 'primary' | 'secondary' | 'tertiary' | 'transparent'
+type Gradient =
+  | 'secondary-to-transparent'
+  | 'accent-to-secondary'
+  | 'accent-to-transparent'
+interface CardSeparatorProps {
```

Comment:
Don't put interfaces at the top of the file, if you only use them once — put them where you use them